go/vt/vtgate/engine/vindex_lookup.go

```diff
@@ -227,10 +227,6 @@ func (vr *VindexLookup) executeBatch(ctx context.Context, vcursor VCursor, ids [
 	} else {
 		result, err = vcursor.ExecutePrimitive(ctx, vr.Lookup, bindVars, false)
 	}
-	if err != nil {
```

Comment:
huh

Reply:
Yeah, the diff is silly but I figured it made more sense to hoist the error return that wrapped the error vs keeping the bare return.